### DISCUSSION TITLE
RSSBrew beta 简单介绍和使用指南 - 1nan

### DISCUSSION THREAD

**Discussion by giscus** [yinan-c/pages-comment#12] [Announcements]
# RSSBrew beta 简单介绍和使用指南 - 1nan

https://yinan.me/rssbrew-config.html

**Comment by flashlab** [comment]
跟freshrss的条件搜索功能比强在哪？

**Reply by yinan-c** [reply]
条件搜索应该指的是这个吧 https://freshrss.github.io/FreshRSS/en/users/10_filter.html
我看了一下，如果是简单用是否包含某一个关键词来过滤的话，freshrss 这个应该够了。
如果想要支持复杂的过滤，比如满足 A 且不满足 B，rssbrew 支持正则表达式和支持 not any all 组合过滤条件的过滤组。理论上应该可以实现更加广的关键词过滤表达式。

**Reply by yinan-c** [reply]
另外如果需要更加实时的更新，rssbrew 支持自定义 cron，freshrss 应该只能每 20 分钟更新一次，不过 rssbrew 本身只是一个处理源的中间件，还是需要配合 rss 阅读器来使用

**Reply by flashlab** [reply]
过滤功能还行，与或非是支持的，但正则不支持(作者明确表达了不会支持)，比较郁闷的是输出的可设置项太少，比如不能按publish date排序，新增加的源永远排在最前面(需求反反复复提了至少5年，依旧没实现)。希望rssbrew如果能支持更多输出设置，比如增删字段，支持网站图标这些 :smirk:

**Reply by yinan-c** [reply]
rssbrew 默认是不管源默认按照 publish date 排序的，如果过滤后想要按照源分组也可以用 digest 功能，不过只能每周每天生成一篇。网站图标可以考虑支持，应该是 parse 就能搞定的事情，增删字段是指？

**Reply by flashlab** [reply]
比如我不想要description字段，增加置顶标记，增加或者限制item数量等等

**Reply by yinan-c** [reply]
> 比如我不想要description字段

这个可以考虑，但不能保证。因为我怕设置项多了反而增加了应用的复杂（现在配置已经很复杂了）
> 增加置顶标记

置顶标记是指在已经过滤的基础上，给特定关键词再标记吗？（也就是给关键词区分重要程度？），如果是这样，你对置顶标记怎么添加有什么想法？

> 增加或者限制item数量等等

这个暂时不会考虑，因为我不想要仅仅通过 item 数量就筛选文章，如果你觉得未读太多可以一刀切得 toggle entries 关闭，只看 weekly/daily digest.

**Reply by flashlab** [reply]
[可以参考这个](https://github.com/hexojs/hexo-generator-feed?tab=readme-ov-file#options)增加自定义模板功能。
